assets/airports/rjss.json

```diff
@@ -0,0 +1,964 @@
+        "OWARI": ["N35d21m06.20", "E136d58m57.86"],
+        "OWLET": ["N37d46m23.00", "E140d27m47.10"],
+        "OYABE": ["N36d36m18.35", "E136d45m12.98"],
+        "PAPNA": ["N36d37m22.00", "E138d01m23.00"],
+        "PERID": ["N38d09m44.80", "E141d07m49.43"],
+        "PRINK": ["N37d46m53.40", "E140d34m25.90"],
+        "QUAIL": ["N37d51m53.70", "E140d34m06.00"],
+        "QUIST": ["N37d47m23.10", "E140d40m59.00"],
+        "RAFUL": ["N32d24m43.00", "E131d06m10.00"],
+        "RAYJO": ["N33d49m48.00", "E136d08m47.00"],
+        "RIBON": ["N37d52m22.20", "E140d40m23.40"],
+        "RIDER": ["N38d19m06.70", "E141d41m14.50"],
+        "RIKYU": ["N37d33m27.80", "E140d27m31.80"],
+        "RUBIS": ["N37d44m51.00", "E140d38m08.00"],
+        "SAITI": ["N39d26m00.10", "E141d15m11.10"],
+        "SAKYU": ["N35d55m01.00", "E134d31m05.00"],
+        "SAMBO": ["N40d11m15.92", "E140d57m01.37"],
+        "SAMON": ["N36d14m34.00", "E134d30m12.00"],
+        "SANOH": ["N36d23m46.00", "E139d36m49.00"],
+        "SASAP": ["N37d10m55.00", "E140d28m24.00"],
+        "SAZMA": ["N31d04m31.00", "E130d18m08.00"],
+        "SDE"  : ["N38d08m18.86", "E140d55m17.34"],
+        "SHIMA": ["N34d18m16.00", "E136d35m19.00"],
+        "SHIPS": ["N38d00m10.40", "E140d39m27.40"],
+        "SHIRO": ["N37d36m78.00", "E140d08m80.00"],
+        "SNOOK": ["N37d59m01.50", "E140d54m51.70"],
+        "SOBOH": ["N32d48m51.00", "E131d24m09.00"],
+        "SONBU": ["N36d11m32.00", "E135d35m03.00"],
+        "SQRUM": ["N33d55m37.00", "E132d04m04.00"],
+        "SS701": ["N38d07m58.50", "E140d50m31.70"],
+        "SS901": ["N38d08m54.60", "E140d59m35.60"],
+        "STEED": ["N37d43m36.20", "E140d59m32.80"],
+        "SWAMP": ["N36d19m14.44", "E140d32m17.02"],
+        "SYE"  : ["N36d00m39.33", "E139d50m21.02"],
+        "TADMI": ["N36d49m02.00", "E139d08m57.00"],
+        "TAPPI": ["N41d08m05.00", "E140d09m54.00"],
+        "TATAM": ["N37d55m50.00", "E138d06m13.00"],
+        "TAVIS": ["N38d57m15.14", "E138d03m15.55"],
+        "TENRU": ["N35d40m11.00", "E138d00m25.00"],
+        "TOBBY": ["N41d55m07.29", "E141d45m35.87"],
+        "TOLEK": ["N39d08m13.00", "E140d43m25.00"],
+        "TOLUM": ["N35d34m24.00", "E137d31m44.00"],
+        "TOPAZ": ["N38d05m16.80", "E141d08m33.50"],
+        "TERAD": ["N37d36m13.22", "E138d28m49.19"],
+        "TSURU": ["N40d55m45.68", "E140d26m36.82"],
+        "URUSI": ["N36d49m09.15", "E137d07m53.52"],
+        "UWE"  : ["N39d37m01.66", "E140d11m12.97"],
+        "VIKET": ["N37d42m05.00", "E139d34m15.00"],
+        "WAPPA": ["N39d42m04.07", "E140d33m05.58"],
+        "YANAI": ["N33d46m23.00", "E131d59m17.00"],
+        "YANAT": ["N37d32m08.00", "E139d43m38.00"],
+        "YAYOI": ["N39d19m10.08", "E139d59m33.76"],
+        "YONEX": ["N38d04m34.29", "E140d17m45.80"],
+        "YOHKO": ["N39d09m05.00", "E139d38m07.00"],
+        "YUCCA": ["N38d09m06.50", "E141d01m32.24"],
+        "ZUWAI": ["N36d23m47.00", "E136d56m59.00"]
+    },
+    "restricted": [
```

Comment:
There seems to be multiple restricted areas within the ACA, is there a particular reason these aren't included?
![RJSS](https://user-images.githubusercontent.com/13125311/86651275-4c799180-bfdb-11ea-9230-230aac496640.PNG)

Reply:
The TOHOKU AREAs are just training areas marked where VFR pilots are just required to contact ATC, so it's not really restricted airspace. The R2 area is most likely pointing at the Fukushima Nuclear Power Plant(s) (maybe a familiar name for foreign people too because of the 2011 disaster) but I haven't included them since it's never mentioned in the JCAB AIP, and I'm utterly confused why Jeppesen has that.

Reply:
Any chance you could send me the document on restricted airspace from the AIP? I'm struggling to log in and I just want to verify whether R2 should be added or not.

Reply:
https://mlit-aisjapan.maps.arcgis.com/apps/webappviewer/index.html?id=4cc18798d43f479a9b1a6c931fee6976

This interactive enroute chart should work without login.

Reply:
That'll do thanks! R2 isn't even shown there so I'm happy :+1: